packages/typespec-ts/test/modularIntegration/generated/azure/core/lro/rpc/generated/src/models/models.ts

```diff
@@ -1,27 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { ErrorModel } from "@azure-rest/core-client";
-import { OperationStatus as CoreOperationStatus } from "@azure/core-lro";
-
 /** Options for the generation. */
 export interface GenerationOptions {
   /** Prompt. */
   prompt: string;
 }
 
-/** Provides status details for long running operations. */
-export interface GenerationResponse {
```

Comment:
probably need to look into this.

Reply:
tcgc doesn't return this model to us. but since this model is used in the LRO polling result, and we are not really using it in the modular layer, and RLC layer are not using getAllModels, I guess it's fine for us, not sure if it's going to be an issue for other language? @tadelesh could you help confirm?

Reply:
it is a refine for tcgc to filtering core models from this pr: https://github.com/Azure/typespec-azure/pull/962